docfx_project/articles/bootstrapping-trust/part-01.encryption-everywhere.md

```diff
@@ -0,0 +1,311 @@
+Ziti software system makes a great example! Let us examine Ziti.
+
+In Ziti, all of the above steps require interacting with a Ziti
+Controller. The Ziti Controller manages the Ziti overlay network. It
+maintains a list of known network services, SDK clients, routers,
+enrollments, policies, and much more! All of these pieces working
+together to create a Ziti Network. A Ziti Network is an overlay network
+\- meaning it creates a virtual network on top of some other network
+referred to as the underlay network.
+
+In the Ziti Network, all network resources are modeled as services in
+the Ziti Controller. All services on a Ziti Network are only accessible
+via the Ziti Network. Network services can be made available via a Ziti
+Network in a variety of manners. The preferred method is embedding the
+Ziti SDK inside of applications and servers as it provides the highest
+degree of Zero Trust security. However, it is also possible to configure
+various overly-to-underlay connections to existing network services via
+"router termination" or a particular type of application with the Ziti
+SDK embedded in it called a "tunneler."
+
+The Ziti Controller also knows about one or more Ziti Routers that form
+a mesh network that can create dynamic circuits amongst themselves.
+Routers use circuits to move data across the Ziti Network. Routers can
+be configured to allow data to enter and exit the mesh. The most common
+entry/exit points are Ziti SDKs acting as clients or servers.
+
+Network clients wishing to attach the network use the Ziti SDK to first
+authenticate with the Ziti Controller. During authentication, the Ziti
+SDK client and Ziti Controller will verify each other. Upon successful
+authentication, the Ziti Controller can provide a list of available
+services to dial (connect) or to bind (host) for the authenticated SDK
+Client. The client can then request to dial or bind a service. If
+fulfilled, a session is associated with the client and service. This new
+session is propagated to the necessary Ziti Routers, and the required
+circuits are created. The requesting client is returned a list of Ziti
+Routers that can be connected to complete the last mile of communication
+between the Ziti Overlay Network and the SDK client.
+
+This set of steps covers the pillars of the Zero Trust model! The Ziti
+Controller and SDK Clients verify each other. The client cannot connect
+to network resources or services until it authenticates. After
+authentication, a client is given the least privilege access allowed by
+only being told about and only being able to dial/bind assigned
+services. It is a Zero Trust overly network!
+
+How did this system come into existence? How do the Ziti SDK client and
+Ziti Controller verify each other? How do the routers and controller
+know to validate each other? How is this managed at scale with hundreds
+o Ziti Routers and thousands of Ziti SDK clients? It seems that this is
```

Comment:
while i enjoy reading "hundreds o Ziti Routers" - it probably should be 'of' :D

Reply:
fix'o'ed